datasets/jll/USAFacts-data/public/Readme.md

```diff
@@ -145,69 +146,152 @@ If you are using data share to get data from a public environment into the custo
-![data share public](../customer/images/data%20share/9.png)
+![data share public](./images/data%20share/9.png)
 
-![data share public](../customer/images/data%20share/10.png)
+![data share public](./images/data%20share/10.png)
 
 
 ### Step 2 - Customer Side
 
 1. Go to Data Share Invitations.
 
-![data share customer](../customer/images/data%20share/11.png)
+![data share customer](./images/data%20share/11.png)
 
 2. Click on **demo_public_share** (or any other name you have provided while deploying at public side).
 
-![data share customer](../customer/images/data%20share/12.png)
+![data share customer](./images/data%20share/12.png)
 
 3. Agree to the terms of use and provide subscription, resource group, data share account and received share name. Click **Accept and configure**.
 
-![data share customer](../customer/images/data%20share/13.png)
+![data share customer](./images/data%20share/13.png)
 
 4. Under the *Datasets* tab, checkmark the dataset and click **Map to target**.
 
-![data share customer](../customer/images/data%20share/14.png)
+![data share customer](./images/data%20share/14.png)
 
 5. Provide the storage account name (the one deployed currently) along with other options and give the Path as **receivedcopy**. Click *Map to target*. *Note* that the Path is the container name and cannot contain capital letters.
 
-![data share customer](../customer/images/data%20share/15.png)
+![data share customer](./images/data%20share/15.png)
 
-![data share customer](../customer/images/data%20share/16.png)
+![data share customer](./images/data%20share/16.png)
 
 6. Now under *Details* tab, click **Trigger snapshot** and then click **Full copy**.
 
-![data share customer](../customer/images/data%20share/17.png)
+![data share customer](./images/data%20share/17.png)
 
-![data share customer](../customer/images/data%20share/18.png)
+![data share customer](./images/data%20share/18.png)
 
 7. Optionally you can enable the snapshot schedule if configured at the public side. For that, checkmark the **Daily** schedule under *Snapshot schedule* tab and click *Enable*.
 
-![data share customer](../customer/images/data%20share/19.png)
+![data share customer](./images/data%20share/19.png)
+
+![data share customer](./images/data%20share/20.png)
+
+## Key Vault Secrets And Access Policies
+
+Azure Key Vault is a cloud service for securely storing and accessing secrets. A secret is anything that you want to tightly control access to, such as API keys, passwords, certificates, or cryptographic keys. Here two secrets are being saved in the key vault with access given to the data factory and to the user whose object id will be provided while creating the deployment (details later):
+
+1. SQL server administrator password
+2. Connection string for the blob storage 
+
+The user whose object id will be provided is given full access to all secrets, keys and certificates inside the key vault. While the data factory is given GET access to any secret in the key vault.
```

Comment:
**Correction**
While the data factory  has the GET access to the secrets in the key vault.